### ISSUE TITLE
Null characters at the ends of vcf files

### ISSUE THREAD

**Issue by hepcat72** [ekg/freebayes#578]
I broke up my freebayes jobs and ran them on a compute cluster.  I concatenated the results (grepping out the headers) and did some post-processing.  I was running into strange behavior in the post-processing and eventually worked out that the end of all my vcf files from the separate region runs had thousands of null characters.  I was able to remove the null characters by piping them through `strings`, but I thought I should probably report this as an issue to see if you can confirm this and fix it.  Here's an example of one of my commands that produced the files with trailing null characters:

```
freebayes -b 146_93A-read.fastq.bam --fasta-reference 208_Homo_sapiens.GRCh38.dna_sm.primary_assembly.fa.fasta --vcf 146_93A-read.f
astq.bam.1_14616-1655697.vcf --report-monomorphic --theta 0.00001 --ploidy 2 -K -i -X -u -n 0 --haplotype-length 0 --min-repeat-size 5 --min-repeat-entropy 0 -m 1 -q 0 -R 0 -Y 0 -
e 1000 -F 0.0 -C 1 -G 1 --min-coverage 0 --min-alternate-qsum 0 --use-duplicate-reads --min-alternate-count 1 --min-alternate-fraction 0 -r 1:14616-1655697 > 146_93A-read.fastq.ba
m.1_14616-1655697.std
```

Here's the version:

```
source=freeBayes v1.2.0-2-g29c4002
```

**Comment by ekg** [comment]
Can you confirm this with v1.3.1? Just check one of the same regions.

**Comment by hepcat72** [comment]
I’ll put that on my list, but I’m off all week and address it when I get back.

**Comment by hepcat72** [comment]
I could not reproduce the issue with the latest version.  Also, I noticed that my STDERR files had nulls as well, so I suspect the issue when running the older version must have had something to do with the SLURM script I used to run it.  However, I used the same slurm script with the new one.  The only difference was that the old one was running under springdale6 and the new one was compiled on a different machine running springdale7.  That's my only guess as to what was causing it.  Either way, the springdale6 nodes on our cluster are all but gone now, so it doesn't matter.  I am closing this issue.
